### DISCUSSION TITLE
New ship: 🟢 GitHub Incident Updates

### DISCUSSION THREAD

**Discussion by tuves** [community/community#146469] [Announcements]
![resolved](https://github.com/user-attachments/assets/d9f40174-cb1a-4a1b-bb41-d2b7ec007324)

At GitHub we pride ourselves in how we communicate effectively across time zones, teams, tools, and of course to our customers. We understand that no product is perfect, and there will be times when there are unsuspected degradations or outages. But in keeping true to our aim to make information as open and accessible as possible, we will now be posting updates and status interruptions in real-time here in the community.

This will give you a centralized thread in Community Discussions for you to share your experiences and find up to date information as it impacts your work. 

### What Can You Expect?

- If an incident occurs and is on our GitHub status page, a discussion will post declaring the incident in the community
- The ability to subscribe to open discussion for real-time updates
- Subsequent updates to post on the incident’s discussion thread
- Declaration of the incident resolving, you will see a marked answer and an image indicating the incident is resolved
- If available: a link to the public incident summary 

Thank you for your patience as we launch this and work out possible kinks!

**Comment by raminaalizadeh** [comment]
Support

**Reply by 0987554412** [reply]
@GitHub

**Comment by VichuA2** [comment]
support

**Comment by Andre601** [comment]
Would it be possible to maybe have a dedicated repository for this, so that one can subscribe to any activity there?
I would love to follow incidents here, but I simply cannot check the community discussion frequently to find and subscribe to incident-related discussions.

One alternative would imo be to have issues enabled, as that would be a different medium to subscribe to and get notified about.
Tho, it has the issues (no pun intended) to have people spam issues, unless you block creation via issue-template chooser, while allowing github-actions (or whatever account posts here) to make new ones...

It's... difficult, now that I think about it. But a dedicated repository would imo be a bit better here...

**Reply by tuves** [reply]
Hi @Andre601,

Thank you for the feedback--I'll pass it along to the teams to discuss. For additional context, the reason we wanted to post updates here in real time was for community members who often open new discussions when they've run into a degradation or outage. This can lead to many splintered conversations, many which cannot be answered or acknowledged by our dedicated staff. Even though updates will be provided in real time here, they will also be posted in [GitHub Status](https://www.githubstatus.com/), which you can subscribe to. Another option is to filter your discussions view by labels, and only viewing "incident" ones. 

Essentially, for those who are already here, we want to make the information easier to access and engage with.

**Reply by huehnerlady** [reply]
I would say it is not necessary to move the discussions to is own repository, as long as there is a feature to subscribe to only the inncident updates for example

Currently you can only watch ALL discussions
![image](https://github.com/user-attachments/assets/5dbd70b7-99ae-444a-afff-614de0237637)

So with the current tools I would agree it would be more usable in a separate repository. The other way is to be able to subscribe for example to specific categories, a feature I know has been asked for since Discussions were introduced.

**Reply by Andre601** [reply]
That's kinda the main issue I have with discussions. No option to only subscribe to specific discussion categories to receive notifications about.
It would really help to have this available.

**Reply by samus-aran** [reply]
> That's kinda the main issue I have with discussions. No option to only subscribe to specific discussion categories to receive notifications about.

💯 IMO this would also be a great addition. I've included a [similar feature request that covers this topic](https://github.com/orgs/community/discussions/3951), feel free to upvote and share your perspective, else create a new discussion!

I can only offer a small alternative using the discussions dashboard on your user profile ( have this bookmarked 🔖 ). 

https://github.com/discussions?discussions_q=repo%3Acommunity%2Fcommunity+label%3A%22Incident+%3Aexclamation%3A%22

![Screenshot 2024-12-18 at 10 33 09](https://github.com/user-attachments/assets/6d96db24-2b23-4c4c-b19e-7b5a9985164c)

**Comment by Ck3555** [comment]
Bonjour, j'ai apprécié votre réponse.Bonjour, j'ai évalué votre réponse. Quel est le processus et le but pour lesquels vous vous inscrivez ici ? C'est la première fois que je m'abonne à une telle application, elle semble vraiment utile et fiable.
Puisqu'il n'y a eu jusqu'à présent rien d'inquiétant concernant la sécurité des informations ou les notifications des compétitions, je vous salue respectueusement, l'estimé officiel. Que se passe-t-il ici et comment ?
________________________________
Gönderen: tuves ***@***.***>
Gönderildi: 11 Aralık 2024 Çarşamba 14:38
Kime: community/community ***@***.***>
Bilgi: Ck3555 ***@***.***>; Manual ***@***.***>
Konu: Re: [community/community] New ship: 🟢 GitHub Incident Updates (Discussion #146469)


Hi @Andre601<https://github.com/Andre601>,

Thank you for the feedback--I'll pass it along to the teams to discuss. For additional context, the reason we wanted to post updates here in real time was for community members who often open new discussions when they've run into a degradation or outage. This can lead to many splintered conversations, many which cannot be answered or acknowledged by our dedicated staff. Even though updates will be provided in real time here, they will also be posted in GitHub Status<https://www.githubstatus.com/>, which you can subscribe to. Another option is to filter your discussions view by labels, and only viewing "incident" ones.

Essentially, for those who are already here, we want to make the information easier to access and engage with.

**Comment by showhand168** [comment]
I SEE."吱吱吱，突次"

**Comment by showhand168** [comment]
I SEE."吱吱吱，突次"

**Comment by Marcono1234** [comment]
Is GitHub staff reading comments on these incident discussions?

For example for a recent incident with pull requests, it seems the incident was not actually fully resolved. See https://github.com/orgs/community/discussions/156245#discussioncomment-12797153.

**Comment by AbrahamRdzSus** [comment]
Me podrían ayuda ya intente haciendo como tres tickests diferentes de soporte, use copilot directamente para que me ayudara incluso trate de mandar correo por que me vulneraron la cuenta, prácticamente le cambiaron el correo (mi cuenta oficial de GitHub) use esta para poder comunicarme aquí, me podrían ayudar
